capone/tests/settings.py

```diff
@@ -19,12 +20,16 @@
 
 TEMPLATE_DIRS = (os.path.join(os.path.dirname(__file__), 'templates'),)
 
+# Namespace test database by the tox environment to allow detox to run tests
+# in parallel.
+_ENVNAME = re.sub(r'\W', '', os.environ.get("TOXENV", ""))
 DATABASES = {
     'default': {
-        'NAME': 'capone_test_db',
+        'NAME': 'caponedb%s' % _ENVNAME,
```

Comment:
This allows running the tests concurrently locally. If it's causing problems, it might make sense to remove it.